wiki/Client/Interface/en.md

```diff
@@ -81,98 +79,110 @@ You can identify the current mode selected by either looking at the icon in the
-- `By Difficulty` - Beatmaps will be sorted from easiest to hardest by star difficulty. *Note that this will split apart mapsets!*
-- `By Length` - Beatmaps will be sorted from shortest to longest by length.
-- `By Rank Achieved` - Beatmaps will be sorted from poorest to best by the highest rank achieved on them.
-- `By Title` - Beatmaps will be sorted alphabetically by the first character of their name.
+| Sort criterion | Description |
+| :-: | :-- |
+| `By Artist` | Beatmaps will be sorted alphabetically by artist. |
+| `By BPM` | Beatmaps will be sorted lowest to highest by their BPM. For maps with changing BPM, the highest will be used. |
+| `By Creator` | Beatmaps will be sorted alphabetically by creator. |
+| `By Date Added` | Beatmaps will be sorted by when they were added, from oldest to newest. |
+| `By Difficulty` | Beatmaps will be sorted by star rating, from easiest to hardest. *Note that this will split apart the difficulties of the beatmap!* |
+| `By Length` | Beatmaps will be sorted by length, from shortest to longest. |
+| `By Rank Achieved` | Beatmaps will be sorted by the highest rank achieved on them, from worst to best. |
+| `By Title` | Beatmaps will be sorted alphabetically by the song title. |
 
 ### Search
 
 ![](img/search-bar.jpg)
 
-*Note: You cannot have the chat console or the options sidebar open if you want to search; otherwise, anything you type will be perceived as chat text or as an options search query.*
+*Note: You cannot have the chat console or the options sidebar open if you want to search. Otherwise, anything you type will be perceived as chat text or as an options search query.*
 
 Only beatmaps that match the criteria of your search will be shown. By default, any search will be matched against the beatmaps' artists, titles, creators, and tags.
 
 In addition to searching these fields, you can use filters to search through other metadata by combining one of the supported filters with a comparison to a value (for example, `ar=9`).
 
 Supported filters:
 
-- `artist`: Name of the artist
-- `creator`: Name of the beatmap creator
-- `ar`: Approach Rate
-- `cs`: Circle Size
-- `od`: Overall Difficulty
-- `hp`: HP Drain Rate
-- `keys`: Number of keys (osu!mania and converted beatmaps only)
-- `stars`: Star Difficulty
-- `bpm`: Beats per minute
-- `length`: Length in seconds
-- `drain`: Drain Time in seconds
-- `mode`: Mode. Value can be `osu`, `taiko`, `catchthebeat`, or `mania`, or `o`/`t`/`c`/`m` for short.
-- `status`: Ranked status. Value can be `ranked`, `approved`, `pending`, `notsubmitted`, `unknown`, or `loved`, or `r`/`a`/`p`/`n`/`u`/`l` for short.
-- `played`: Time since last played in days
-- `unplayed`: Shows only unplayed maps. A comparison with no set value must be used. The comparison itself is ignored.
-- `speed`: Saved osu!mania scroll speed. Always 0 for unplayed maps or if the [Remember osu!mania scroll speed per beatmap](/wiki/Client/Options#gameplay) option is off
+| Filter | Description |
```

Comment:
relevant:

![image](https://user-images.githubusercontent.com/36758269/151120844-17965e6c-74b6-437d-b80a-4f4e214df6e7.png)

Reply:
for sanity's sake i'd leave it as just "catch". i don't believe we will support "catchthebeat " going forward, and no one in their right mind would type that if they can just type "catch".

Reply:
Sorry about snubbing this, I kept instinctively checking this off my notifications list thinking it was the 70 odd other review comments bumping this pull rather than something that I actively needed to look at. (These pulls should really be smaller, you know.)

Aside from the other review comments, on a quick look two filters appear to be missing:

* `title` - textual, for the name of the track
* `difficulty`, textual, for the name of a difficulty in a mapset

Rest seems fine.

As for the point re: ruleset filter (the discussion link doesn't even open anything for me) - yes, a prefix match is used, any unambiguous prefix will work. There is also support for several alternative variants (e.g. it appears that stuff like `o!m` will also work as mania). But I think adding that info is too much, I'd say to stick what's already here.

For what it's worth prefix match is also used for map status.